src/lib/utils/sqlite_wrapper.cpp

```diff
@@ -89,9 +284,11 @@ void SQLiteWrapper::create_table(const Table& table, const std::string& table_na
   const auto insert_into_str = insert_into_stream.str();
 
   sqlite3_stmt* insert_into_statement;
-  const auto sqlite3_prepare_return_code = sqlite3_prepare_v2(
-      _db, insert_into_str.c_str(), static_cast<int>(insert_into_str.size() + 1), &insert_into_statement, nullptr);
-  Assert(sqlite3_prepare_return_code == SQLITE_OK, "Failed to prepare statement: " + std::string(sqlite3_errmsg(_db)));
+  const auto sqlite3_prepare_return_code =
+      sqlite3_prepare_v2(main_connection.db, insert_into_str.c_str(), static_cast<int>(insert_into_str.size() + 1),
+                         &insert_into_statement, nullptr);
+  Assert(sqlite3_prepare_return_code == SQLITE_OK,
+         "Failed to prepare statement: " + std::string(sqlite3_errmsg(main_connection.db)));
 
   // Insert
```

Comment:
Extend: `// Insert values column-wise`

Reply:
Isn't it row-by-row?

Reply:
Following line made me think of column-wise inserts.
```cpp
for (auto column_id = ColumnID{0}; column_id < table.column_count(); column_id++) {
```
But you're right. It missed the prepared statement logic.